command/results_xunit.go

```diff
@@ -140,6 +140,7 @@ func (c *xunitResults) parseAndUploadResults(ctx context.Context, conf *model.Ta
 				// logs are only created when a test case does not succeed
 				test, log := tc.toModelTestResultAndLog(conf.Task)
 				if log != nil {
+					log.Lines = append(log.Lines, "system-out:", suite.SysOut, "system-err:", suite.SysErr)
```

Comment:
should we only append if the values are populated? 

can we add some integration tests?

Reply:
added an integration test

Reply:
what about only adding these lines if they're populated?

Reply:
Done